traitcuration/traits/datasources/ols.py

```diff
@@ -7,8 +7,11 @@
 
 from retry import retry
 
+from ..models import Status, OntologyTerm
+
+
 logging.basicConfig()
-logger = logging.getLogger('ZOOMA')
+logger = logging.getLogger('OLS')
```

Comment:
FYI it is a common practice to just always use `logger = logging.getLogger(__name__)` in order to avoid explicitly specifying module name each time. This is not mandatory to adopt, but can be useful